pkg/config/config_template.yaml

```diff
@@ -1502,19 +1502,30 @@ api_key:
 
   ## @param peer_service_aggregation - bool - default: false
   ## @env DD_APM_PEER_SERVICE_AGGREGATION - bool - default: false
-  ## [BETA] Enables `peer.service` aggregation in the agent. If disabled, aggregated trace stats will not include `peer.service` as a dimension.
+  ## DEPRECATED - please use `peer_tags_aggregation` instead.
+  ## Enables `peer.service` aggregation in the agent. If disabled, aggregated trace stats will not include `peer.service` as a dimension.
   ## For the best experience with `peer.service`, it is recommended to also enable `compute_stats_by_span_kind`.
   ## If enabling both causes the Agent to consume too many resources, try disabling `compute_stats_by_span_kind` first.
   ## If the overhead remains high, it will be due to a high cardinality of `peer.service` values from the traces. You may need to check your instrumentation.
   ## NOTE: If you are using an OTel tracer it's best to have both enabled because client/producer spans with a `peer.service` value
   ## may not be marked by the Agent as top-level spans.
   # peer_service_aggregation: false
 
+  ## @param peer_tags_aggregation - bool - default: false
+  ## @env DD_APM_PEER_TAGS_AGGREGATION - bool - default: false
+  ## [BETA] Enables aggregation of peer related tags (e.g., `peer.service`, `db.instance`, etc.) in the agent.
+  ## If disabled, aggregated trace stats will not include these tags as dimensions on trace metrics.
+  ## For the best experience with it is recommended to also enable `compute_stats_by_span_kind`.
+  ## If enabling both causes the Agent to consume too many resources, try disabling `compute_stats_by_span_kind` first.
+  ## If the overhead remains high, it will be due to a high cardinality of peer tags from the traces. You may need to check your instrumentation.
+  ## NOTE: If you are using an OTel tracer it's best to have both enabled because client/producer spans with a `peer.service` value
```

Comment:
```suggestion
  ## NOTE: If you are using an OTel tracer, it's best to have both enabled because client/producer spans with a `peer.service` value
```